### ISSUE TITLE
[Bug]: HTML Preview based on confDir not srcDir

### ISSUE THREAD

**Issue by mgrosvenor** [vscode-restructuredtext/vscode-restructuredtext#402]
### Expected behavior

The configuration directory (`confDir`) is not always at the root of the document tree. This is why it is provided as a configuration option (and is distinct from `srcDir`). However, the preview generator still makes this [assumption](https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/5f028453217681410f5b5abfefb60be51d4a8e93/src/preview/rstEngine.ts#L43). The expected behaviour is that preview generation will depend on `srcDir` instead. 

### Actual behavior

If the `confDir` is not at the root of the document tree, incorrect paths are generated and previewer fails with an error. 

### Detailed steps

build a document tree as follows:
```
/index.rst
/docs/index.rst
/docs/example.rst
/docs/conf.py
```

Try to preview `example.rst` it will fail.

### Extension information

189.1.0 (but also fails on 190)

### Visual Studio Code information

1.70.2

### Operating system information

macOS

### Python information

3.8

### Sphinx information

5.2

### Contents of conf.py

```python
N/A
```

### Relevant log output

```shell
N/A
```

**Comment by AlexanderLanin** [comment]
Same here. By accident I can report, that there are some combinations that will work as well, something like confDir is ".." from buildDir and srcDir is ".." from confDir. But I'm not sure which conditions needs to be fulfilled exactly. It's best to fix this :-)

As already mentioned, what helps for now is:
put conf.py into srcDir and set confDir = srcDir.

**Comment by mgrosvenor** [comment]
> put conf.py into srcDir and set confDir = srcDir.

Unfortunately this is not an option for me. This is how I found the bug. 

The team I'm working with has put the conf.py in a place different to the src folder. They have reasons for doing this. I cannot get them to change their reasons simply because my editor doesn't play nicely :-( 

I've proposed a fix for this issue in PR #403.

**Comment by lextm** [comment]
Starting from version 190.1.17, this extension no longer performs live preview but recommends the Esbonio extension to be used instead. Close this issue now.
